app/server/utils/gristify.ts

```diff
@@ -95,7 +96,7 @@ export class Gristifier {
    * Remove all Grist metadata tables. Warning: attachments are considered metadata.
    */
   public async degristify() {
-    const db = await SQLiteDB.openDBRaw(this._filename);
+    const db = await SQLiteDB.openDBRaw(this._filename, OpenMode.OPEN_READONLY);
```

Comment:
Regarding my last comment, this is the line that puzzled me. I wonder if this change is intended, at this location (instead of at line 130).

Please tell me when I can review again, so I don't rush :)

Reply:
Ooops! :) Good catch.